idx/query.go

```diff
@@ -51,32 +49,34 @@ func NewRegexpQuery(field, regexp []byte) (Query, error) {
 }
 
 // NewNegationQuery returns a new query for finding documents which don't match a given query.
-func NewNegationQuery(q Query) (Query, error) {
```

Comment:
maybe include the error return to be nice to users of the API for now?  We should have the m3ninx+m3db+m3coordinator monorepo sometime next week, then you'll be to change all call-sites more easily.

Reply:
I think it would be easier to just rip the band-aid off now. I can update m3db and m3coordinator once this lands.

Reply:
sounds good